packages/cli/src/redwood-tools.js

```diff
@@ -89,86 +103,232 @@ export const copyFiles = async (src, dest) => {
   fixProjectBinaries(getPaths().base)
 }
 
+const rwtCopy = ({ RW_PATH = process.env.RW_PATH }) => {
+  RW_PATH = resolveFrameworkPath(RW_PATH)
+
+  console.log(`Redwood Framework Path: ${c.info(RW_PATH)}`)
+
+  const src = `${RW_PATH}/packages/`
+  const dest = `${getPaths().base}/node_modules/@redwoodjs/` // eslint-disable-line
+
+  copyFiles(src, dest)
+}
+
+const rwtCopyWatch = ({ RW_PATH = process.env.RW_PATH }) => {
+  RW_PATH = resolveFrameworkPath(RW_PATH)
+
+  console.log(`Redwood Framework Path: ${c.info(RW_PATH)}`)
+
+  const src = `${RW_PATH}/packages/`
+  const dest = `${getPaths().base}/node_modules/@redwoodjs/`
+
+  chokidar
+    .watch(src, {
+      persistent: true,
+      recursive: true,
+      ignored: [path.join(src, 'packages/create-redwood-app/template')],
+    })
+    .on(
+      'all',
+      _.debounce((event) => {
+        // TODO: Figure out if we need to only run based on certain events.
+        console.log('Trigger event: ', event)
+        copyFiles(src, dest)
+      }, 500)
+    )
+}
+
+const rwtLink = async (yargs) => {
+  const RW_PATH = yargs.RW_PATH || process.env.RW_PATH
+  const { clean } = yargs
+
+  if (!RW_PATH) {
+    console.error(c.error('You must specify a path to your local redwood repo'))
+    process.exit(1)
+    return
+  }
+
+  const frameworkPath = resolveFrameworkPath(RW_PATH)
+
+  console.log(`Redwood Framework Path: ${c.info(frameworkPath)}`)
+
+  // Check if /redwood included in workspaces
+  const pkgJSONPath = path.join(getPaths().base, 'package.json')
+  const packageJSON = require(pkgJSONPath)
+  const isRedwoodInWorkspaces = packageJSON.workspaces?.packages.some(
+    (workspacePath) => workspacePath.match(/redwood\/?\*?/)
+  )
+
+  if (!isRedwoodInWorkspaces) {
+    console.log(`You don't have redwood in your workspace package`)
+    const { shouldAddWorkspacePath } = await prompts({
+      type: 'confirm',
+      name: 'shouldAddWorkspacePath',
+      message:
+        'Add redwood folder to your workspaces config? \n This will modify your package.json',
+    })
+
+    if (shouldAddWorkspacePath) {
+      const updatedPackageJSON = {
+        ...packageJSON,
+        workspaces: {
+          packages: [...packageJSON.workspaces.packages, 'redwood/*'],
+        },
+      }
+
+      fs.writeFileSync(
+        pkgJSONPath,
+        prettier.format(JSON.stringify(updatedPackageJSON), {
+          parser: 'json-stringify',
+        })
+      )
+
+      console.log('🎉 Done! Workspaces now has redwood/')
+    } else {
+      console.log('Cancelling contributor workflow...')
+
+      return
+    }
+  }
+
+  const packagesPath = path.join(frameworkPath, 'packages')
+
+  console.log(`Linking your local Redwood build from ${c.info(packagesPath)}`)
+
+  await execa(`ln -sf ${packagesPath} ./redwood`, {
+    shell: true,
+    stdio: 'inherit',
+    cwd: getPaths().base,
+  })
```

Comment:
@dac09 This is the solution for the double packages/packages
```suggestion
  if (!fs.statSync('./redwood').isDirectory()) {
    await execa(`ln -sf ${packagesPath} ./redwood`, {
      shell: true,
      stdio: 'inherit',
      cwd: getPaths().base,
    })
  }

```

Reply:
Nice!